spec/config/mongoid.rb

```diff
@@ -2,12 +2,14 @@
   config.connect_to('delayed_job_shallow_mongoid_test')
 end
 
+Mongo::Logger.logger = ::Logger.new('tmp/mongo.log') if defined?(Mongo)
+
 RSpec.configure do |config|
   config.before(:each) do
     Mongoid.purge!
     Mongoid::IdentityMap.clear if Delayed::ShallowMongoid.mongoid3?
   end
   config.after(:all) do
-    Mongoid.default_session.drop
+    Mongoid.default_session.drop if Mongoid.respond_to?(:default_session)
```

Comment:
+1 to replicating the compatibility approach from [here](https://github.com/mongoid/mongoid-slug/blob/master/spec/spec_helper.rb#L61). I'd rather not add a new dependency (even a development dependency just for these 1 or 2 lines.